### DISCUSSION TITLE
間違った内容や古くなった内容を、著者の確認や同意が無くても修正できる方がQiitaの目的に沿っている

### DISCUSSION THREAD

**Discussion by shakakibi** [increments/qiita-discussions#352] [要望・アイデア・不具合]
間違った内容や古くなった内容を著者の確認や同意が無いと修正できないのは、コミュニティガイドラインのエンジニアにとって再利用性・汎用性の高い情報が集まる場を作るというQiitaの目的に反している。
著者がもうQiitaをやめていたり、エンジニアとしての学習をやめてしまっていたり、Qiitaでの活動ができない状況だったりするとQiitaにどんどん古い技術の負債が積み重なって、いつか古い情報の墓場と化してしまう。
間違った内容や古くなった内容を誰でも削除できるようにすべきとは言わないが、せめて修正は著者の確認や同意なく誰でもできる場であってほしい。

**Comment by Verclene** [comment]
そういうのはWikiタイプのプラットフォームですればいい気がします.
QiitaやZennのようなブログ型プラットフォームで第三者の積極的な編集を許すと,文脈が取っ散らかるとか記事の題意にそぐわない内容になるのは目に見えています.
それによって情報の再利用性や汎用性が下がってしまうのであれば本末転倒と言うものです.

古い情報を書き換える必要性についてはある程度同意しますが,やむを得ず古いバージョンを使わなければならない場合に情報が残っていないと困るでしょうし,
例えば記事で取り上げている技術自体がDiscontinuedな状態で,最新の代替技術を挙げてまで補完する必要があるか？と聞かれるとそんなに需要はないと思います.

そもそもライブラリの正しい使い方が知りたいとかいうレベルで汎用性や検証可能性をとにかく追及するのであれば,ネット記事を参照する前に公式ドキュメントを参照するのが暗黙のルールというものでは？

>削除できるようにすべきとは言わないが、せめて修正は

個人的な見解では,削除よりも修正のほうが重大な問題であるように感じます.
記事を外部から参照できないならまだしも,引用当時から第三者によって意図せず内容が書き換わっているという状況を是としてよいものでしょうか?

いずれにせよ,利用規約上最もクリティカルな著作権帰属のルールを変えないといけないので,
権利尊重をガイドラインとして掲げるQiitaにおいて第三者が直接編集できるようにすることには強く反対します.

**Comment by shiracamus** [comment]
コメントに「この記事の情報は古く、最新版ではこうなっています」と書いてはいかがでしょうか？

**Comment by munierujp** [comment]
自分の場合は、基本的に編集リクエストを送るようにしています。
とはいえアカウントが明らかにアクティブではなく編集リクエストの取り込みが期待できない場合は、@shiracamusさんの言うようにコメント欄に書くというのはいいアイディアだと思います。
読者がそこまでちゃんと読んでくれるかという問題はありますが……

**Comment by JunichiIto** [comment]
僕も古い内容や間違った内容はコメント欄で指摘するようにしてるんですが、いかんせん、コメントってあまり目立たないんですよね。。
初心者の人はコメント欄まで到達せずに「よっしゃ、わかったぞ！」って言って離脱してしまいそうな気がしています。

なので、僕個人としてはコメントを目立たせてほしいんですが、記事の炎上を招きやすくなる、といった理由であまり目立たせてもらえないようです😢

参考 https://github.com/increments/qiita-discussions/discussions/234

**Comment by tanaka-asami415** [comment]
@shakakibi  みなさま
たくさんのご意見をいただきありがとうございます🙏

運営としては、第三者が勝手に内容を編集できるようにすることは考えておりませんが
記事への編集リクエストや、コメントがあることが気づかれやすいような改善をすることにより
再利用性・汎用性の高い情報が集まる場となるようにしていきたいと思っております。
直近の開発優先度などから、お時間をいただいてしまいますが、お待ちいただけると幸いです🙇
